src/services/plugin.service.ts

```diff
@@ -16,21 +16,38 @@ export class PluginService {
 
     const reference = (this.plugins[pluginName] = new PluginLoader(pluginName, args) as IPlugin);
 
-    this.registerCommands(pluginName);
+    this.registerAliases(pluginName, args);
 
     return reference;
   }
 
-  registerCommands(pluginName: string, args?: any): void {
-
-    const commands = this.plugins[pluginName].pluginCommands;
-
-    if (!commands.includes(pluginName)) {
-      this.commands[pluginName] = pluginName;
+  registerAliases(pluginName: string, args?: any): void {
+
+    const aliases = this.plugins[pluginName].pluginAlias;
+
+    if (aliases === undefined) {
+      this.aliases[pluginName] = pluginName;
+    } else {
+      if (!aliases.includes(pluginName)) {
+        if (this.aliases[pluginName] !== undefined) {
+          args.loggerService.warn(
```

Comment:
I know you think this is sketchy, but I think it is a valid fail-safe assuring all plugins that they can at least be called by the name of the plugin.